test/libsolidity/syntaxTests/types/struct/typehash_member_userdefined.sol

```diff
@@ -0,0 +1,10 @@
+type T is uint256;
+contract C {
+    struct S {
+        T t;
+    }
+
+    bytes32 h = type(S).typehash;
```

Comment:
See comment above, we should allow this IMO.

Reply:
Fixed this